src/prefect/utilities/logging.py

```diff
@@ -131,7 +131,9 @@ def emit(self, record) -> None:  # type: ignore
             record_dict = record.__dict__.copy()
 
             # ensures emitted logs respect configured logging level
-            config_level = getattr(logging, prefect.context.config.logging.level, 20)
+            config_level = getattr(
+                logging, prefect.context.config.logging.level, "INFO"
```

Comment:
Sorry, I meant `logging.INFO` instead of `20`, not `"INFO"`. I'm actually surprised this doesn't error below.

```suggestion
                logging, prefect.context.config.logging.level, logging.INFO
```